dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/cluster/WorkerClusters.java

```diff
@@ -59,27 +62,44 @@ public Optional<WorkerServerMetadata> getServer(final String address) {
         return Optional.ofNullable(workerMapping.get(address));
     }
 
-    public List<String> getWorkerServerAddressByGroup(String workerGroup) {
+    public List<String> getDbWorkerServerAddressByGroup(String workerGroup) {
+        if (WorkerGroupUtils.getDefaultWorkerGroup().equals(workerGroup)) {
+            return UnmodifiableList.unmodifiableList(new ArrayList<>(workerMapping.keySet()));
+        }
```

Comment:
I am not sure if we still need to keep `default` logic here.
In the past, the default worker group is a logic worker group, it will contain all worker.
Right now, if a worker configures its worker group at config e.g. spark_worker_group, then we may don't need to auto put it at default group.

Reply:
I think we should keep the default value of worker group. If users don't want this default value, they can modify it to other custom values.